client/landing/stepper/declarative-flow/internals/steps-repository/site-migration-already-wpcom/style.scss

```diff
@@ -0,0 +1,66 @@
+@import "@wordpress/base-styles/breakpoints";
+
+.site-migration-already-wpcom__form-title--error,
+.site-migration-already-wpcom__form-error {
+	color: var(--color-error);
+	font-size: 0.875rem;
+	font-weight: 500;
+}
```

Comment:
Just curious, are we intentionally avoiding using scss way for selectors here? like using `&` etc so that, for example, it's easier to find styles when searching with selectors?

Reply:
Yes, I received some feedback to avoid them for debugging purposes.